### PR TITLE
docs: Do not  use class property arrow functions for event handlers

### PR DESCRIPTION
**Pull request by ptbrowne**
No more class properties for handlers as they are difficult to mock.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,23 +85,36 @@ class MyComponent extends Component {
 
 ## Binding event handlers
 
-Avoid binding event handlers in `constructor`, leverage `transform-class-properties`
+Binding event handlers in `constructor`, do not leverage `transform-class-properties`
```
Comment (y-lohse) [inline README.md]:
```suggestion
Bind event handlers in `constructor`, do not leverage `transform-class-properties`
```

**Comment by ptbrowne** [comment]
> I guess that's a lot of code to migrate

It's only a guideline and can be done on a case by case basis, where you need it, mostly for tests ;)

**Comment by CPatchane** [comment]
Are they not supposed to be transpiled by Babel before being used by Jest? I thought this was resulting to the same code but I was wrong then :/

**Comment by ptbrowne** [comment]
It does not result in the same code.

VIa babel.io

```
class Toto {
  a = () => {}
}
```

```
"use strict";

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }

var Toto = function Toto() {
  _classCallCheck(this, Toto);

  this.a = function () {};
};
```

**Comment by CPatchane** [comment]
Ok then, thanks for this precision @ptbrowne 👍

**Pull request merged**